log-viewer/modules/parsers/LineParser.ts

```diff
@@ -278,8 +289,9 @@ class CodeUnitStartedLine extends LogLine {
         break;
       default:
         this.cpuType = "method";
-        if (name && name.startsWith("VF:"))
+        if (name && name.startsWith("VF:")) {
```

Comment:
I think JS has optional chaining these days.
This should work.
```suggestion
        if (name?.startsWith("VF:")) {
```